src/main/java/collectors/RegionalHouseholdStats.java

```diff
@@ -129,7 +158,45 @@ public void record() {
                     homelessAnnualisedTotalIncome += h.getMonthlyPreTaxIncome();
                 }
             }
+            // check if the househld is a commuter travelling from one region to another
+            House home=h.getHome();
+            Region region=h.getRegion();
+            if(region!=null && home !=null){
+            	if(home.getRegion().getRegionId()!=region.getRegionId()){
+            		this.nCommuter++;
+            	}
+            }
+            // sum the monthly travl cost of each household who need to commute
```

Comment:
Yes, it will be more efficient to move the following two calculations into the previous ifs.
However, I thought it would be easier to extend the module later by considering the travel cost and time in the region with the current codes.

Reply:
Do you mean extending the model to take into account some kind of "intra-region" commuting costs?

Reply:
Yes. I am not sure if you have a plan to further extend the model in such a way?

Reply:
Ok, yeah, taking into account a non-zero cost for intra-region commuting sounds like a reasonable idea. I'll write it down for possible future updates.